_posts/2024-03-01-pyos-newsletter-march-2024.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: single
+title: "pyOpenSci Newsletter: March 2024"
+excerpt: "New partnerships, packages, and conference announcements!"
+author: "Jesse Mostipak"
+permalink: /blog/pyos-newsletter-march-2024.html
+header:
+  overlay_color: "#666"
+  overlay_filter: 0.6
+categories:
+  - blog-post
+  - community
+toc: true
+comments: true
+---
+
+## :star: Welcome, AstroPy
+
+pyOpenSci has officially partnered with the [AstroPy Project](https://www.astropy.org/) through our [Community Partnership program](https://www.pyopensci.org/partners.html). AstroPy is a Community Python Library for Astronomy, and the AstroPy Project is a community effort to develop a single core package for Astronomy in Python, while also fostering interoperability between Python astronomy packages. We're thrilled to be partnering with the AstroPy project, and can't wait to share more!
+
+
+## :purple_heart: Hello, sunPy
+
+[sunPy](https://sunpy.org/), a Python package that provides fundamental tools for accessing, loading, and interacting with solar physics data in Python, has successfully gone through the pyOpenSci open peer review process, which is the first step in our budding partnership with the heliophysics community!
+
+You can read more about sunPy and pyOpenSci in [this lovely blog post](https://sunpy.org/posts/2024/2024-01-24-pyopensci) from Nabil Friej.
+
+## :wave: Say hi to sciform
+
+The [sciform package](https://sciform.readthedocs.io/en/stable/), which is used to convert Python numbers into strings according to a variety of user-selected scientific formatting options, was accepted into the pyOpenSci ecosystem! sciform, like all of the pyOpenSci packages, completed the [pyOpenSci open peer review process](https://www.pyopensci.org/about-peer-review/index.html). And the community support around getting the sciform package review-ready was amazing to see!
```

Comment:
Two questions here Jesse.

1. can we link to the author / submitor github profile here to give them more visibility. Justin? we could even include a picture of him. BUT please tell em if i'm making a low hanging fruit task heavier to implement. 
2. You have created these awesome package graphics for social. i wonder if we created them with this newsletter, then you could include the graphic in the post and then have it ready to go later when you promote the package. 
thoughts?

Reply:
@lwasser links are super easy! I've added in Justin's GH repo. and I agree with you on the package graphics for socials - I'll get those starting in May. when we "launched" this newsletter back in March I don't believe we'd started the social graphics for packages!

Reply:
ok that sounds like a perfect plan. so when we accept new packages we can create a graphic and then promote it later on social whenever the timing fits (in between promoting the already-accepted packages. does that sound inline with what you are thinking? i'm just thinking when we accept a package, that triggers a graphic. and then we have some backlog of packages we haven't promoted yet too.

Reply:
exactly! so right now I've been working through the backlog of existing packages and promoting those on socials, but we'll eventually be caught up (yay!) and so what I'd like to eventually have is something like:
* package is accepted into pyOS
* I make a graphic and get the package promoted on socials within two weeks of the package being accepted
* the image (and package announcement) will get rolled into the newsletter (one day we'll have a full section JUST for new packages each month!)
* and in a perfect world, a package developer writes a guest blog post for the pyOS website 🤞